.env.dev

```diff
@@ -14,12 +14,13 @@ DJANGO_SUPERUSER_PASSWORD=Password123
 DJANGO_SUPERUSER_EMAIL=csf@csf.com
 DJANGO_SUPERUSER_USERNAME=admin
 
-EMAIL_HOST=mailhog
-EMAIL_PORT=1025
-EMAIL_HOST_USER=''
-EMAIL_HOST_PASSWORD=''
-EMAIL_USE_TLS=False
-EMAIL_ADDRESS_FROM=email-tester@test.com
+EMAIL_HOST='smtp.sendgrid.net'
+EMAIL_PORT=587
+EMAIL_HOST_USER='apikey'
+EMAIL_USE_TLS=True
+EMAIL_ADDRESS_FROM='test@codersforcauses.com'
+SENDGRID_API_KEY=''
+
```

Comment:
this needs to stay as the mailhog version for dev. I will update the production env file